cmd/root.go

```diff
@@ -165,6 +165,10 @@ func AddCommands(cmd *cobra.Command) {
 		cfg.API.PublicListener, "Socket for grpc services that are save to expose publicly.")
 	cmd.PersistentFlags().StringVar(&cfg.API.PrivateListener, "grpc-private-listener",
 		cfg.API.PrivateListener, "Socket for grpc services that are not safe to expose publicly.")
+	cmd.PersistentFlags().StringVar(&cfg.API.PostListener, "grpc-local-listener", cfg.API.PostListener,
+		"Socket for grpc services that are not safe to expose publicly, but can be accessible within a local network")
```

Comment:
It's confusing, why a `PostListener` takes 
>services that are not safe to expose publicly, but can be accessible within a local network

? Shouldn't it be something like:
```suggestion
	cmd.PersistentFlags().StringVar(&cfg.API.PostListener, "grpc-post-service-listener", cfg.API.PostListener,
		"address for the GRPC post-service server to listen on")
```

Reply:
Yeah, an oversight from back when the listener was still intended to be used as an in-between of `private` and `public`, where it can only be exposed on private cidrs - I'll improve the description.